Data-Structures/src/test/java/datastructures/tree/BinaryTreeTest.java

```diff
@@ -0,0 +1,159 @@
+package datastructures.tree;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class BinaryTreeTest {
+    @Test
+    public void testPreOrder() {
+        BinarySearchTree searchTree = new BinarySearchTree();
+        BinaryTree tree = new BinaryTree();
+
+        searchTree.addNode(5);
+        searchTree.addNode(8);
```

Comment:
You should make a seperate test file to test your `BinarySearchTree` class. You are testing it in the `BinaryTreeTest`